### ISSUE TITLE
Asset identifier event for sub parts of the period

### ISSUE THREAD

**Issue by RufaelDev** [Dash-Industry-Forum/AdInsertion#84]
Current asset identifier is on the period level, for some burned in 
ads or changes within the period this is not sufficient, 
an event scheme with equivalent signaling is proposed to mark the asset identification of those parts: 

     
    <EventStream
      schemeIdUri="urn:smpte:ul:060E2B34.01040101.01200900.00000000"
      timescale="1">
      <Event
        presentationTime="1615563936"
        duration="30"
        id="26">
        "USTR0000003H"
      </Event>
     </EventStream>

In the absence of other identification schemes, a DASH-IF defined scheme may be 
signalled by setting: @schemeIdUri to "http://dashif.org/guidelines/v5/asset-id"
@value to a MovieLabs ContentID URN ([58], 2.2.1)

**Comment by mokojo200** [comment]
Sounds like a way of solving this issue especially for Barker channels , and clients that do not handle Multi period that well

**Comment by haudiobe** [comment]
Makes sense - should  be implemented somewhere. A proposal would be nice @RufaelDev 

We should also permit to change Assets whenever there is a virtual Period event - this should be added to the Amd.2 of MPEG-DASH @haudiobe.

**Comment by olivier-yospace** [comment]
Hi @haudiobe ,
The SVTA Advertising Working Group work in the SVTA2053: Ad Creative Signaling in DASH and HLS document should cover this.
The new scheme is named: “urn:svta:advertising-wg:ad-creative-signaling” and will allow to provide the start, duration and id of every ads in the period.

**Comment by haudiobe** [comment]
Should be added to updated guidelines.
